### PR TITLE
Adds TWTGradient files and updates podspec

### PR DESCRIPTION
**Pull request by dfowjtm**
@prachigauriar @jnjosh @macdrevx

### CODE REVIEW

File: UIKit/Gradient/TWTGradient.h [hunk 1]
```diff
@@ -0,0 +1,104 @@
+//
+//  TWTGradient.h
+//
+//  Created by Prachi Gauriar on 11/17/2015.
+//  Copyright © 2015 Ticketmaster Entertainment, Inc. All rights reserved.
+//
```
Comment (prachigauriar) [inline UIKit/Gradient/TWTGradient.h]: Needs license.

File: UIKit/Gradient/TWTGradient.m [hunk 2]
```diff
@@ -0,0 +1,205 @@
+//
+//  TWTGradient.m
+//
+//  Created by Prachi Gauriar on 11/17/2015.
+//  Copyright © 2015 Ticketmaster Entertainment, Inc. All rights reserved.
+//
```
Comment (prachigauriar) [inline UIKit/Gradient/TWTGradient.m]: Needs license.

File: UIKit/Gradient/TWTGradient.m [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+
+- (instancetype)initWithHexadecimalRGB:(NSUInteger)hex location:(CGFloat)location
+{
+    CGFloat red = (CGFloat)((hex & 0xff0000) >> 16) / 255.0;
+    CGFloat green = (CGFloat)((hex & 0x00ff00) >> 8) / 255.0;
+    CGFloat blue = (CGFloat)(hex & 0x0000ff) / 255.0;
+    UIColor *color = [UIColor colorWithRed:red green:green blue:blue alpha:1.0];
+    return [self initWithColor:color location:location];
+}
```
Comment (prachigauriar) [inline UIKit/Gradient/TWTGradient.m]: This seems out of place. We have the color category already. Can we get rid of this?

Comment (dfowjtm) [inline UIKit/Gradient/TWTGradient.m]: Yes

File: UIKit/Gradient/TWTGradient.h [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+- (instancetype)initWithColor:(UIColor *)color location:(CGFloat)location NS_DESIGNATED_INITIALIZER;
+
+/*!
+ @abstract Convenience initializer that initializes a new gradient color with a hexadecimal RGB value.
+ @discussion The color is assumed to have an alpha component of 1.0.
+ @param hex The new instance’s color as a hexadecimal RGB value in 0xRRGGBB format.
+ @param location The new instance’s location. Values of this parameter are pinned to be between 0.0 and 1.0, inclusive.
+ @result A newly created TWTGradientColor instance with the specified color and location.
+ */
+- (instancetype)initWithHexadecimalRGB:(NSUInteger)hex location:(CGFloat)location;
```
Comment (prachigauriar) [inline UIKit/Gradient/TWTGradient.h]: This doesn’t feel right given that we have the color category in this project anyway.

Comment (dfowjtm) [inline UIKit/Gradient/TWTGradient.h]: Removed

**Comment by prachigauriar** [comment]
👍 Squash before merging.

**Comment by prachigauriar** [comment]
Let’s get another review, since I wrote the original code :-)

**Comment by macdrevx** [comment]
@dfowjtm @prachigauriar I'd love to see some unit tests.

**Comment by macdrevx** [comment]
But in lieu of unit tests, 👍

**Comment by prachigauriar** [comment]
@dfowjtm Let’s get this merged.

**Pull request merged**